custom_models/mental_health/models/mental_health_notes.py

```diff
@@ -11,3 +11,4 @@ class MentalHealthNotes(models.Model):
     icd10_codes = fields.Selection(string='ICD10 Code', required=False, copy=False,
                                    selection=[('f40.00', 'F40.00 - Agoraphobia unspecified'),
                                               ('f40.01', 'F40.01 - Agoraphobia with panic disorder')])
+    homework = fields.Text(string='Homework Assignment')
```

Comment:
Do we think this should be required or copyable?

Reply:
I don't think it should be required because it seems possible that a user might want to have a note without one. I think it should be copyable since the homework assignment likely wouldn't change week to week. If "copy" is left out, will it default to being copyable?

Reply:
It defaults to true unless you set the copy field to false! I agree with it not being required too. Let me QA it and we should be good to go